pkg/providers/instance/instance.go

```diff
@@ -166,6 +167,11 @@ func (p *Provider) Delete(ctx context.Context, id string) error {
 		}
 		return fmt.Errorf("terminating instance, %w", err)
 	}
+	for _, instance := range response.TerminatingInstances {
+		if instance.CurrentState.Name != aws.String(ec2.InstanceStateNameTerminated) {
+			return cloudprovider.NewRetryableError(fmt.Errorf("terminating instance '%s'", aws.StringValue(instance.CurrentState.Name)))
```

Comment:
How ofter are we logging this error to customers? Also this doesn't seem like an error, but more debug information